app/src/main/java/com/example/houta/dadvice_v01/MainActivity.java

```diff
@@ -81,9 +117,13 @@ private void addNativeExpressAds() {
 
         // Loop through the items array and place a new Native Express ad in every ith position in
         // the items List.
-        for (int i = 0; i <= mRecyclerViewItems.size(); i += ITEMS_PER_AD) {
-            final NativeExpressAdView adView = new NativeExpressAdView(MainActivity.this);
-            mRecyclerViewItems.add(i, adView);
+        int i;
+        for (i = 0; i <= mRecyclerViewItems.size(); i=i+1){;//i += ITEMS_PER_AD) {
+            if((i % ITEMS_PER_AD) == 0) {
+                final NativeExpressAdView adView = new NativeExpressAdView(MainActivity.this);
+                mRecyclerViewItems.add(i, adView);
+            }
+
```

Comment:
Different implementation of ads that is irrelevant to pull request but does not change functionality